missions/Controllers/Demo2/ActivityPredictionViewController.swift

```diff
@@ -129,3 +147,144 @@ extension ActivityPredictionViewController {
 //        })
     }
 }
+
+// Implemented by Elina
+
+@available(iOS 13, *)
+extension ActivityPredictionViewController {
+    
+    func setUpButtons() {
+        
+        setUpBackwardButton()
+        setUpForwardButton()
+        
+    }
+    
+    @IBAction func playButtonTapped(_ sender: UIButton) {
+        
+        if AudioPlayback.shared.audioPlayer.currentTime == AudioPlayback.shared.audioPlayer.duration {
+            AudioPlayback.shared.audioPlayer.currentTime = 0.00
+        } else {
+            self.playbackState = self.playbackState.toggle
+            self.playButton.setImage(self.playbackState.image, for: .normal)
+            updateRunLoop()
+        }
+        
+        AudioPlayback.shared.play(pan: stanceSlider.value)
+      
+      }
+    
+    @IBAction func onStanceChanged(_ sender: UISlider) {
```

Comment:
Optional: for any new method or action you write, make sure to add some documentation by clicking the line above the method declaration and pressing CMD+/. It will automatically create a document string for you to fill in and describe the purpose of the method.